app/assets/stylesheets/tpi/_colors.scss

```diff
@@ -1,5 +1,7 @@
 $white: #fff;
+$blue-light: #86A9F9;
 $blue: #5587F7;
+$blue2: #2465F5;
```

Comment:
Don't use numbers for colors. The more verbose the better to understand where to use.

Reply:
Yeah, I know, couldn't find a good name for a color between blue and blue-dark :D

Reply:
Normally I use this http://chir.ag/projects/name-that-color/

Reply:
maybe we should use it for all colors?

Reply:
Sure. Was just a comment, not a blocker :-)